src/functions/acalc.h

```diff
@@ -44,6 +49,9 @@ inline QTime blocktime(QTime tofb, QTime tonb)
  */
 inline QString minutesToString(QString block_minutes)
 {
+    DEB << "#######################################";
+    DEB << "# WARNING: USE OF DEPRECATED FUNCTION #";
+    DEB << "#######################################";
```

Comment:
We can avoid polluting debug out put with a simple deprecation macro
```c++
QT_DEPRECATED
inline QSTring minutesToString(...)
```

Reply:
I added the QT_DEPRECATED makro, very helpful!